registry/quilt_server/const.py

```diff
@@ -36,3 +36,4 @@ def blacklisted_name(username):
 PASSWORD_RESET_SALT = 'reset'
 MAX_LINK_AGE = 60 * 60 * 24 # 24 hours
 CODE_EXP_MINUTES = 10
+TOKEN_EXP_DEFAULT = {'days': 90}
```

Comment:
minor: this is not an "expiry" default but a "time to live" or "lifetime" default. expiry varies by token.